From: Kir Kolyshkin <kolyshkin@gmail.com>
Date: Thu, 5 Jul 2018 11:03:52 -0700
Subject: [PATCH] aufs: fix Wrapf args

Fix the following go-1.11beta1 build error:

> daemon/graphdriver/aufs/aufs.go:376: Wrapf format %s reads arg #1, but call has 0 args

While at it, change '%s' to %q.

Signed-off-by: Kir Kolyshkin <kolyshkin@gmail.com>
Signed-off-by: Sebastiaan van Stijn <github@gone.nl>
Origin: upstream, https://github.com/moby/moby/commit/2e30e9e
---
 daemon/graphdriver/aufs/aufs.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/engine/daemon/graphdriver/aufs/aufs.go b/engine/daemon/graphdriver/aufs/aufs.go
index 915225277017..ac2fe26ecd5c 100644
--- a/engine/daemon/graphdriver/aufs/aufs.go
+++ b/engine/daemon/graphdriver/aufs/aufs.go
@@ -373,7 +373,7 @@ func atomicRemove(source string) error {
 	case os.IsExist(err):
 		// Got error saying the target dir already exists, maybe the source doesn't exist due to a previous (failed) remove
 		if _, e := os.Stat(source); !os.IsNotExist(e) {
-			return errors.Wrapf(err, "target rename dir '%s' exists but should not, this needs to be manually cleaned up")
+			return errors.Wrapf(err, "target rename dir %q exists but should not, this needs to be manually cleaned up", target)
 		}
 	default:
 		return errors.Wrapf(err, "error preparing atomic delete")
